gateway/webhook/webhook.go

```diff
@@ -247,6 +248,27 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 				continue
 			}
 
+			// begin
+			if parseAllQueryParams {
```

Comment:
I don't know if we have source information at this level, but if we have, I would drive it through a configuration that is set only to "Shopify" by default as we don't need this for all sources.

Reply:
I think the information we have at this point is just that it's a webhook type source.

Reply:
we could have some information by parsing the payload being sent the webhook URL maybe. But again for that, we are parsing anyway. So not sure on this one.

Reply:
@cisse21 any thoughts?

Reply:
@saikumarrs  `breq.sourceType` . Here, we have the information re sourceType (here it is Shopify). So do we want to add a check and only parse query params for `breq.sourceType == "Shopify"` ? Will that work?